Makefile

```diff
@@ -138,7 +143,7 @@ update-nvml-h:
 # Generate an image for containerized builds
 # Note: This image is local only
 .build-image: Dockerfile
-	$(DOCKER) build \
+	$(DOCKER) buildx build \
```

Comment:
What motivated the switch to `buildx`?

Reply:
`ARG	TARGETARCH`

Is not available with the normal build

Reply:
I think it's fine to assume that buildx is available since this is only used to generate the bindings (and not something end users will ever need to run).